src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -162,8 +171,10 @@ const HoloFuelDnaInterface = {
       return noDuplicateIds.filter(tx => tx.status === 'pending').sort((a, b) => a.timestamp < b.timestamp ? -1 : 1)
     },
     decline: async transactionId => {
+      const transaction = await HoloFuelDnaInterface.transactions.getPending(transactionId)
       await createZomeCall('transactions/decline')({ origin: transactionId })
       return {
+        ...transaction,
         id: transactionId,
         status: STATUS.rejected,
         direction: DIRECTION.incoming
```

Comment:
The direction could actually be incoming or outgoing, as it is possible to decline both a request and an offer.  Let's just maintain the direction value returned in the 'transaction' object.

NB: Direction in this context indicates the final directional displacement of the funds, not the direction of a transaction stage.  (ie. A spender could be the recipient of a request, but not the recipient of the funds.)

Reply:
ah, good catch, thanks.